src/waits.ts

```diff
@@ -12,7 +12,7 @@ import { getValue, getElement, getValueWait, getConditionWait } from './transfor
  * @example I wait until 'Search Bar > Submit Button' to be clickable (timeout: 3000)
  */
 When(
-    'I wait until {string} {playwrightConditionWait}{playwrightTimeout}',
+    'I wait until {string} {playwrightConditionWait}( ){playwrightTimeout}',
```

Comment:
why do you decide to add optional space in the step definition instead of parameter type regexp?

Reply:
VSCode extension showed such steps as undefined. Wanted to keep the best UX for end users